### ISSUE TITLE
Failed installing: libvorbis

### ISSUE THREAD

**Issue by Airamek** [pspdev/psptoolchain#104]
What I did:
1. clone the repo
2. ./prepare-mac-os.sh
3. sudo ./toolchain-sudo.sh
4. Fix issues with fixes described under https://github.com/pspdev/psptoolchain/issues/77 (I used this: "https://github.com/pspdev/psptoolchain/compare/master...iaDRM:xissue_72" to fix opentri) 
5. Retry building

After these steps only 1 error remained, libvorbis.
Please help!

[psptoolchain_buildlog.txt](https://github.com/pspdev/psptoolchain/files/2710833/psptoolchain_buildlog.txt)

**Comment by carstene1ns** [comment]
It looks to me like libogg has built fine, but later the libvorbis examples could not link to it. Can you upload the `config.log` from libvorbis folder to verify which libogg was found (thinking it might wanted to use the host's)?

**Comment by Airamek** [comment]
Hi, sorry for responding this late, here is the libvorbis config.log
[config.log](https://github.com/pspdev/psptoolchain/files/2730461/config.log)

**Comment by carstene1ns** [comment]
It is as I figured:
```
pkg_cv_OGG_CFLAGS=-I/usr/local/Cellar/libogg/1.3.3/include
pkg_cv_OGG_LIBS='-L/usr/local/Cellar/libogg/1.3.3/lib -logg'
```

Seems we need to bend pkg-config configuration or provide our own wrapper (as other toolchains already do).

**Comment by carstene1ns** [comment]
This one is fixed since #130 (and even before since https://github.com/pspdev/psplibraries/commit/e7126df5ffda9e304e2a0b27cdd60a6c4b150ebc).
